fe/src/pages/HelloWorld.vue

```diff
@@ -3,17 +3,30 @@
   <ReviewForm />
   <ReviewRoom />
   <ReviewHost />
+<<<<<<< Updated upstream
```

Comment:
there are still conflicts, please fix

Reply:
sorry i forgot to remove this file. will remove this file since I remove this file on the other PR's maam since there is already a file that contains all components